_includes/getting-started.html

```diff
@@ -1,106 +1,124 @@
-    <div id="getting-started">
-    <div style="margin-top: auto;" id="box"> 
-      <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
-        with Meshery
-      <p style="font-size: 20px;">Choose a platform and see how easy it is to get up and running with a service mesh.</p></h1>
-    </div>
-  </div> 
-<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -250px;">
-<div class="text editable code">
+<div id="getting-started">
+  <div style="margin-top: auto;" id="box">
+    <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
```

Comment:
<img width="303" alt="Screenshot 2020-04-28 at 9 12 33 PM" src="https://user-images.githubusercontent.com/44091822/80507979-14c7eb00-8995-11ea-84b1-407ed0a18e32.png">
Can this be fixed ?

Reply:
may you check again? I think now this issue is solved.